assets/javascript/app.js

```diff
@@ -0,0 +1,92 @@
+
+  var ingredient = ["eggs+", "apples+", "rice"];
+  var queryUrl = "https://spoonacular-recipe-food-nutrition-v1.p.rapidapi.com/recipes/findByIngredients?number=10&ingredients=" + ingredient;
+  
+
+  var popularIngredients = ["chicken", "steak", "pork", "salt", "cumin", "tomatoes", "eggs", "milk", "rice", "cheese", "chicken broth", "tomato paste", "rolled oats", "nuts", "worcestershire sauce", "flour", "garlic", "onions", "oregano", "cinnamon", "cayenne", "black pepper", "kosher salt", "yogurt", "oil", "vingear", "potatoes", "syrup", "beans", "lemons", "tuna", "butter", "bell peppers", "sugar"]
+  // var popularMeats = [""];
+
+  // var popularVegetables = [""];
+
+  // var popularFruits = [""];
+
+  // var popularDairy = [""];
+
+  // var popularSpices = [""];
+
+function displayIngredient(){
+  $("#ingredientsection").empty();
+
+  for (var i =0; i<popularIngredients.length; i++){
+    var newButton = $("<button>").text(popularIngredients[i]);
+      //gives the button a class of new button
+          newButton.addClass("ingredients");
+          //gives the data attribute of data-name
+          newButton.attr("data-name", popularIngredients[i]);
+          //puts button in button section
+          $("#ingredientsection").append(newButton);
+    }
+  }
+
+
+  $("#add-ingredient-button").on("click", function(event){
+
+    event.preventDefault();
+    var newIngredient = $("#ingredient-input").val().trim();
+    popularIngredients.push(newIngredient)
+    
+    displayIngredient()
+    $("#ingredient-input").val("");
+
+  })
+
+displayIngredient();
+
+ $("#submit-button").on("click", function(){
+
+  displayRecipe()
+
+});
+
+
+function displayRecipe(){
+console.log("it worked")
+// var movie = $(this).attr("data-name");
+
+$.ajax({
+  url: "https://spoonacular-recipe-food-nutrition-v1.p.rapidapi.com/recipes/findByIngredients?number=5&ranking=1&ignorePantry=false&ingredients=apples%2Cflour%2Csugar",
+  headers: { 
+    'X-RapidAPI-Host': 'spoonacular-recipe-food-nutrition-v1.p.rapidapi.com',
+    'X-RapidAPI-Key': '98db7bca28msh8d5ab6517d13d17p17dbecjsn45b21b9d6bda'},
+  method: "GET"
+}).then(function(response) {
+  console.log(response);
+
+
```

Comment:
were we not using the API to define the popular ingredients and Recipes automatically?
Also Is there a reason for all the empty lines? if your doing it to show where to put other code, id suggest instead just using a commented line and leave lines above and below it empty. This way we can just inject code into that one line. This also could just be personal pref as everyone has their own way of coding. someone else can chime in if im wrong.